Lombiq.ChartJs/Models/DataLabelAlignmentConfiguration.cs

```diff
@@ -1,14 +1,15 @@
-using Newtonsoft.Json;
 using System;
+using System.Text.Json.Serialization;
 
 namespace Lombiq.ChartJs.Models;
 
+[JsonSourceGenerationOptions(PropertyNamingPolicy = JsonKnownNamingPolicy.CamelCase)]
```

Comment:
Looks like STJ just doesn't have the kind of per-class casing definition that we are used to from Newtonsoft. The technique you linked is what I was trying to avoid, because you have to pass the serialization context to `JsonSerializer.Serialize()` as a second parameter. I will have to get rid of all `[JsonSourceGenerationOptions(PropertyNamingPolicy = JsonKnownNamingPolicy.CamelCase)]` and replace them with `[JsonProperty("nameButCamelCase")]` for every property. tragic.